### DISCUSSION TITLE
Merging my account names

### DISCUSSION THREAD

**Discussion by ddehilster** [community/community#22616] [New to GitHub]
<p>I have read the page for merging accounts but I really am not seeing how to do it. I get the repository part. But I have committs under two different account names. What do I do to change the commits to another account name so I can delete the others?</p>

**Comment by Greek** [comment]
<p>Not sure if you can do that…</p>

**Comment by ThirdEyeRose** [comment, marked as answer]
<p>Hi <a class="mention" href="/ddehilster">@ddehilster</a>,</p>
<p>Your commits are linked to your email address. If you remove the email address from your old account and add it to the new account on GitHub, it should associate the right GitHub username.</p>
<p>Let me know if that works for you or if I can be of further help.</p>
<p>Thanks!</p>

**Comment by ajeshkaushik99** [comment]
<p>dear my email is not merging for other accont</p>

**Comment by ajeshkaushik99** [comment]
<p>how much will given pay ment for this work</p>

**Comment by ajeshkaushik99** [comment]
<p>i am not using this mail id only for u</p>
